### DISCUSSION TITLE
How to display extra information in BagPart's item list

### DISCUSSION THREAD

**Discussion by KP-WhatEver** [OrchardCMS/OrchardCore#11755] [Q&A]
I have a bag with custom content items with a taxonomy field. Now I want to display the taxonomy's value without having to open all the bag item (basically show the taxonomy value in the list).

Question: What is the best way to go? Do I need to override a template? (If so, what name?)
Thanks for any hints :)

**Comment by ns8482e** [comment]
Content Item within the `BagPart` is rendered with `DisplayType` based on BagPart's settings. Default is `Summary`

You can override a template  `[ContentType]_[DisplayType]` for the content type that you use for BagPart Items

**Reply by KP-WhatEver** [reply]
Thank you for your answer. 
I need to clarify my question a bit; I want to display additional values in the editor view of the bag. 
Is this possible with something like the pattern of the AliasPart? Or do i have to override the template and copy mostly the source and add my little extra info?
